src/cascadia/TerminalControl/TerminalControl.vcxproj

```diff
@@ -10,6 +10,16 @@
     <SubSystem>Console</SubSystem>
     <!-- sets a bunch of Windows Universal properties -->
     <OpenConsoleUniversalApp>true</OpenConsoleUniversalApp>
+    <!-- C++/WinRT sets the depth to 1 if there is a XAML file in the project -->
+    <!--
+         Unfortunately for us, we need it to be 3. When the namespace merging

```

Comment:
these seem like they could be one big block comment instead of two separate comments.